app/models/entity.rb

```diff
@@ -0,0 +1,8 @@
+class Entity < ApplicationRecord
+  belongs_to :group
```

Comment:
-  [ ] Entities should also  have_many groups, this is a project requirement, kindly check the provided ERD diagram. Their relationship should be many to many 
![image](https://github.com/ivanmvh/rails_wallet/assets/62705947/6689f123-4bfc-4d82-90bd-e86af97a2fd3)